### ISSUE TITLE
Is there a way to have line breaks after each record?

### ISSUE THREAD

**Issue by iamrahulroy** [michaeldv/awesome_print#207]
To be clear I'm aware of **multiline option**, but that's not what I want(I would like to keep that option disabled). So what do I want?
Let's suppose I'm fetching a collection of users, I want a line breaks after each user record. Something like this:  

![example file](https://cloud.githubusercontent.com/assets/2527366/10560891/2a20bb30-7536-11e5-82ff-8d520589940d.png)

*PS: Let me know if this is not the right place to ask this question.*

**Comment by waldyr** [comment]
Thanks for using awesome_print.

>  I want a line breaks after each user record.

Currently this is not possible for awesome_print. But we are looking into ways that you would declare how you want things to be printed.

As a workaround you could do something like:
![screen shot 2016-06-19 at 1 02 36 pm](https://cloud.githubusercontent.com/assets/1893997/16178441/24f8f31c-361e-11e6-8191-86f0400d8eed.png)

> PS: Let me know if this is not the right place to ask this question.

This is indeed the right place, feel free to ask! :)

Anymore questions?

**Comment by iamrahulroy** [comment]
> But we are looking into ways that you would declare how you want things to be printed.

A simple toggle setting would be really helpful something like `insert_linebreaks: true`. Anyways, thanks for response @waldyr ;) 

_PS: Feel free to close this issue._

**Comment by waldyr** [comment]
> A simple toggle setting would be really helpful something like insert_linebreaks: true.

Unfortunately this options seems to be too specific, however I might be wrong and if you provide a PR I will definitely review it. @gerrywastaken Wdyt?

> thanks for response @waldyr ;)

You're welcome. Sorry for the late response. Almost one year of waiting 😖

**Comment by gerrywastaken** [comment]
@waldyr @iamrahulroy I agree a PR would make adding a change far more likely. If there was a change where the added complexity to functionality ratio was good and it didn't break anything, that would make merging it much easier. I can see how having a separator between elements would boost readability in many cases, so I like the idea.

Also, sorry you had to wait so long. @waidyr only recently joined and has already been doing a fantastic job of going through the backlog of tickets.

**Comment by iamrahulroy** [comment]
@waldyr @gerrywastaken Don't worry about the response time. I understand how open source works. The fact that you guys are responding to backlog issues is commendable. 

Coming back to PR thing, I would be more than happy to look into it if you guys can point me in the right direction. Though I can't promise anything as I'm pretty sure most of the codebase is filled with Metaprogramming magic 😅

**Comment by waldyr** [comment]
> Coming back to PR thing, I would be more than happy to look into it if you guys can point me in the right direction. 

Here are the directions:
- How you add a new option: https://github.com/awesome-print/awesome_print/pull/245
- Where you want to add logic for new options: [Array Formatter](https://github.com/waldyr/awesome_print/blob/239c990a1325f403f93112ff872e31c21f6c8784/lib/awesome_print/formatters/array_formatter.rb)
